elifesciences/eLife-JATS-schematron
# improvement: various updates

- Fix message text in `head-subj-test1`
- Remove `head-subj-test-2` and fix `head-subj-test-1`
- Add some tests for author xrefs - `author-xref-tests`
- Disallow text in aff elements - `gen-aff-test-6`
- chore: whitespace cleanup in `err-elem-cit-high-2-2` for readability
- Add missing GitBook reference page links
- Add `elife-check` - reverse of `elife-ref-check`
